src/seedu/addressbook/data/person/Name.java

```diff
@@ -42,7 +42,12 @@ public static boolean isValidName(String test) {
     public List<String> getWordsInName() {
         return Arrays.asList(fullName.split("\\s+"));
     }
-
+	
+	public List<String> getWordsInNameLowerCase() {
+		String fullNameLowerCase = this.fullName.toLowerCase();
+		return Arrays.asList(fullNameLowerCase.split("\\s+"));
```

Comment:
Nice simple solution to make the find function case insensitive. Love it!

Reply:
Good initiative, didn't think of it!